src/python/ddapp/startup.py

```diff
@@ -450,6 +451,9 @@ def fitPosts():
     mappingDemo = mappingdemo.MappingDemo(robotStateModel, playbackRobotModel,
                     ikPlanner, manipPlanner, footstepsDriver, atlasdriver.driver, lHandDriver, rHandDriver,
                     perception.multisenseDriver, view, robotStateJointController, playPlans)
+    if useMappingPanel: #  if false, might break task panel TODO: handle
```

Comment:
I fixed this, removed the comment and tested that it works for LWR, Val2, and V5